apps/data-pipeline/degreeworks-scraper/src/components/Scraper.ts

```diff
@@ -192,6 +161,7 @@ export class Scraper {
       Cookie: `X-AUTH-TOKEN=${authCookie}`,
       Origin: "https://reg.uci.edu",
     };
+    console.log(headers);
```

Comment:
This should be removed to prevent leaking the user's token.

Reply:
lmao oops i was trying to run manual requests